test/includes/microcloud.sh

```diff
@@ -1146,7 +1156,13 @@ create_system() {
       exec > /dev/null
     fi
 
-    lxc init ubuntu-minimal-daily:24.04 "${name}" --vm -c limits.cpu=4 -c limits.memory=4GiB
+    lxc init "ubuntu-minimal-daily:${os}" "${name}" --vm -c limits.cpu=4 -c limits.memory=4GiB -c security.devlxd.images=true
+
+    # Pre fetch additional images to be used by the VM through security.devlxd.images=true
+    lxc image copy ubuntu-minimal-daily:24.04 local:
+    lxc image copy ubuntu-minimal-daily:22.04 local:
+    lxc image copy ubuntu-minimal-daily:24.04 local: --vm
+    lxc image copy ubuntu-minimal-daily:22.04 local: --vm
```

Comment:
LGTM but I'd probably do the pre-fetching before the first `lxc init`.

Reply:
Agree, fixed.